docs/source/changelog.rst

```diff
@@ -14,69 +14,80 @@ Changelog
 .. +++++++++
 
 
-v0.14.0 / 2020-MM-DD
+v0.14.0 / 2020-02-06
 --------------------
 
 New Features
 ++++++++++++
+- (:pr:`212`) NWChem - Adds CI for the NWChem harness.
+- (:pr:`226`) OpenMM - Moves the OpenMM harness to a canonical forcefield based method/basis language combination.
+- (:pr:`228`) RDKit - Adds MMFF94 force field capabilities.
 
 Enhancements
 ++++++++++++
--(:pr:`215`) Psi4 - indicating location by either PATH or PYTHONPATH sets up both subprocess and API execution.
--(:pr:`215`) ``get_program`` shows the helpful "install this" messages from ``found()`` rather than just saying "cannot be found".
+- (:pr:`201`) Psi4 - ``psi4 --version`` collection to only grab the last line.
+- (:pr:`202`) Entos - Adds wavefunction parsing.
+- (:pr:`203`) NWChem - Parses DFT empirical dispersion energy.
+- (:pr:`204`) NWChem - Allows custom DFT functionals to be run.
+- (:pr:`205`) NWChem - Improved gradient output and added Hessian support for NWChem.
+- (:pr:`215`) Psi4 - Psi4 location can be found by either PATH or PYTHONPATH sets up both subprocess and API execution.
```

Comment:
Psi4 - if Psi4 location can be found by either PATH or PYTHONPATH, harness sets up both subprocesses and API execution